packages/packherd-require/src/sourcemap-support.ts

```diff
@@ -226,18 +212,8 @@ class SourcemapSupport {
       state.curPos = pos
       frame = cloneCallSite(frame)
 
-      const originalFunctionName = frame.getFunctionName
-
-      frame.getFunctionName = function () {
-        if (state.nextPos == null) {
-          return originalFunctionName()
-        }
-
-        return state.nextPos.name ?? originalFunctionName()
-      }
-
       frame.getFileName = function getFileName () {
```

Comment:
These changes are still necessary to get accurate line numbers for typescript files in development